src/server/scripts/Outland/Auchindoun/ManaTombs/boss_pandemonius.cpp

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license: https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-GPL2
+ * Copyright (C) 2008-2016 TrinityCore <http://www.trinitycore.org/>
+ * Copyright (C) 2005-2009 MaNGOS <http://getmangos.com/>
+ */
+
+#include "ScriptMgr.h"
+#include "ScriptedCreature.h"
+#include "Player.h"
+
+enum Texts
+{
+    SAY_AGGRO           = 0,
+    SAY_KILL            = 1,
+    SAY_DEATH           = 2,
+    EMOTE_DARK_SHELL    = 3
+};
+
+enum Spells
+{
+    SPELL_VOID_BLAST = 32325,
+    SPELL_DARK_SHELL = 32358
+};
+
+enum Events
+{
+    EVENT_VOID_BLAST = 1,
+    EVENT_DARK_SHELL
+};
+
+
+class boss_pandemonius : public CreatureScript
+{
+public:
+    boss_pandemonius() : CreatureScript("boss_pandemonius") { }
+
+    CreatureAI* GetAI(Creature* creature) const
+    {
+        return new boss_pandemoniusAI(creature);
+    }
+
+    struct boss_pandemoniusAI : public ScriptedAI
+    {
+        boss_pandemoniusAI(Creature* creature) : ScriptedAI(creature)
+        {
+        }
```

Comment:
What is this good for?

Reply:
is a problem?

Reply:
No I just wonder what the use of empty brackets are ^^

Reply:
are the brackets of the constructor

Reply:
Maybe just put them inline like : 
https://github.com/azerothcore/azerothcore-wotlk/pull/3165/files/083b3c942ff8f26024d68b6324ef48c77b9f2dbe#diff-83ffc1c83be6eb3b673c01afcfc12e20R35

I find this better for (my) eyes hahaha

Reply:
ok